src/views/homepage/components/learn-section/index.tsx

```diff
@@ -51,17 +52,15 @@ export default function LearnSection({
           className={s.tutorialsList}
           style={
             {
-              '--tutorials-count': tutorials.length,
+              '--tutorials-count': collectionCards.length,
```

Comment:
⛏️ Nit: Perhaps we should update this css var? `--collection-cards-count`?  (non blocking)

Reply:
Good point! Glazed right over this one 😆